### DISCUSSION TITLE
NIGEL NO MORE - What happoens to WPCLOUDdeploy now :(

### DISCUSSION THREAD

**Discussion by ghost** [WPCloudDeploy/wp-cloud-deploy#102] [General]
<img width="1640" alt="image" src="https://github.com/WPCloudDeploy/wp-cloud-deploy/assets/98357389/4241ae7b-8dbe-4166-b415-0bc95a68fbd6">

https://www.herdegenfuneralhome.com/obituaries/Nigel-Rasheed-Bahadur?obId=31617195

Nigel Bahadur no more ... its shocking. May his soul rests in eternal peace. I wonder how would the WPCLOUDdeploy survive now :(

**Comment by qvdrei** [comment]
Damn, it really shocked me! May god rest him in peace. I would want to take over the project but I have access only to the ALL ACCESS BUNDLE. Does anyone have the files for the Multi-Tenanacy Module and Momentum?

**Reply by frebecchi** [reply]
I'm shocked as well. May he rest in peace. Unfortunately, I think it might be difficult to get access to Momentum. The product was still relatively new, and unless we find someone who purchased the license and has full access to the files/source, I'm afraid the project might be lost. I also only have access to the ALL ACCESS BUNDLE, and unfortunately, both Momentum and the Multi-tenant Module are not included in the All Access package.

**Reply by qvdrei** [reply]
I can't believe Nigel didn't have anyone on the team... He was a genius if he was able to do all the products by himself (and help from contractors)

**Reply by ghost** [reply]
All this while Nigel continued to enjoy the simple pleasures of life like dancing and all. He truly was a genuis.

**Reply by qvdrei** [reply]
@SuperHondo yees.. I saw a lot of comments about dancing on his funeral page.. just one about WPCD and most of them about how good of a dancer he was, haha

**Comment by jaiivarsson** [comment]
This is very sad, and I hope Nigel's family and friends are ok.
Would happily help in anyway to keep this project alive too. 
Also, my downloads off some of the extensions are out of date and I would love to be able to download the latest of all of them.

**Reply by qvdrei** [reply]
> This is very sad, and I hope Nigel's family and friends are ok. Would happily help in anyway to keep this project alive too. Also, my downloads off some of the extensions are out of date and I would love to be able to download the latest of all of them.

Give me an email, and I will send you a drive link.

**Reply by jaiivarsson** [reply]
Thanks @qvdrei can you email gentle.rain1296@4amail.com please?

**Reply by ghost** [reply]
> Give me an email, and I will send you a drive link.

thekingkongrules @ gmail(.)com - lets me also get some helping @qvdrei Thanks

**Reply by qvdrei** [reply]
Sent! To both of you! Let's gather a group of people and work towards developing this product.

**Reply by MaqAnquor** [reply]
> Sent! To both of you! Let's gather a group of people and work towards developing this product.

were you able to get the Multi-Tenanacy Module and git module?
 i remember he had aws script also! share your email id and will share the plugins i have...

**Reply by qvdrei** [reply]
@nightcrawler123 

That would be amazing.

Here's my email:

andreidemoexpert(at)gmail(dot)com

**Reply by ghost** [reply]
> > Sent! To both of you! Let's gather a group of people and work towards developing this product.
> 
> were you able to get the Multi-Tenanacy Module and git module? i remember he had aws script also! share your email id and will share the plugins i have...

@nightcrawler123 if you could also Cc me the same ---  thekingkongrules @ gmail(.)com

**Reply by MaqAnquor** [reply]
> thekingkongrules @ gmail(.)com

done, were you able to get the other plugins.

**Reply by ghost** [reply]
> > thekingkongrules @ gmail(.)com
> 
> done, were you able to get the other plugins.

Many thanks yes, I did get the link. I had access to till WPCD was live - ALL ACCESS PASS LTD. Sadly with Nigel's passing I couldnt even get to discuss the AI Momentum with him

**Reply by qvdrei** [reply]
@nightcrawler123 I got only the all acces bundle (which I already have). Did you also share the multi-tenancy, git module and I missed something?

here's my email again in case you need it.

andreidemoexpert(@)gmail(dot)com

Thanks

**Reply by MaqAnquor** [reply]
@qvdrei no i only got the plugins i use from the all access bundle. i do not have git and other

**Reply by jaiivarsson** [reply]
Thank for sending through the link. 

Would be good to have the documentation too but for now at least there is the way back machine on [archive.org](https://web.archive.org/web/20231124035306/https://wpclouddeploy.com/doc-landing/)

**Reply by qvdrei** [reply]
Yep. I'm thinking on building an website similar to Nigel's and add the documentation there. But first, I want to have Nigel's family/team approval that I can take over the project. If he was alone, then probably no one has access to his work which is truly sad.

**Reply by MaqAnquor** [reply]
@qvdrei I called his mother, sent her whatsapp, there is no response.it would be better if we fork and rename the product.

**Reply by qvdrei** [reply]
@nightcrawler123 

Can you also give me his mother's phone number, I'll try my luck. You can send it here. 

andreidemoexpert(@)gmail(dot)com

**Reply by MaqAnquor** [reply]
WPCD: Warning - certain critical CRONS are not running on your site. Below are the ones that appear to be missing:

scan_scheduled_servers

Has anybody faced this error recently?

**Reply by ghost** [reply]
> @nightcrawler123
> 
> Can you also give me his mother's phone number, I'll try my luck. You can send it here.
> 
> andreidemoexpert(@)gmail(dot)com

I feel we should not get so greedy to disturb a Mother in grief. She just lost her son and few back she lost her duaghter. I feel we should be more humane in our approach. I had spoken once with her to express thoughts in her moment of grief. Beyond which there wasn't anything she knew on Nigel's work life. Sad but true.Rest is your or anyone else's call.

**Reply by ghost** [reply]
> WPCD: Warning - certain critical CRONS are not running on your site. Below are the ones that appear to be missing:
> 
> scan_scheduled_servers
> 
> Has anybody faced this error recently?

Deactivate the WPCD Plugin, count till ten and Reactivate the WPCD. This realigns the CRON schedules, it should help you.

**Reply by MaqAnquor** [reply]
Done it multiple times not populating cannot find this cron

**Reply by MaqAnquor** [reply]
if you guys are thinking of supporting wpcd can you please spped up server deployment currently it takes ages.... at least on linode.

**Reply by ghost** [reply]
> if you guys are thinking of supporting wpcd can you please spped up server deployment currently it takes ages.... at least on linode.

It will take minimum of 45mins on Linode / Vultr... nothing that WPCD can help around which.

**Reply by MaqAnquor** [reply]
well if we use custom server and virtual plugin the server provisioning is very fast in comparision

**Reply by ghost** [reply]
> well if we use custom server and virtual plugin the server provisioning is very fast in comparision

Time taken never bothered me much frankly speaking - For me DigitalOcean has always worked very fast and above that I have stuck to using BYOS which takes the time it takes but normally under 10mins give or take.

**Reply by MaqAnquor** [reply]
thats true. DO is fast. but linode is too too slow.

**Reply by drbyrd** [reply]
Hi All,

I'm in a similar boat. I've been trying to access the site for a few days when I stumbled on the news about Nigel passing. This is such upsetting news! Nigel was amazing, so passionate about the products, and Momentum was coming along so well. It's a shame.

If someone like @nightcrawler123 would be so kind as to email me the addons and extras, I'd be so grateful! You can reach me at  [REDACTED].

**Reply by MaqAnquor** [reply]
added you to my gdrive 

> Hi All,
> 
> I'm in a similar boat. I've been trying to access the site for a few days when I stumbled on the news about Nigel passing. This is such upsetting news! Nigel was amazing, so passionate about the products, and Momentum was coming along so well. It's a shame.
> 
> If someone like @nightcrawler123 would be so kind as to email me the addons and extras, I'd be so grateful! You can reach me at [at]gmail[dot]com.

**Reply by drbyrd** [reply]
@nightcrawler123 Thank you kindly. Do you know if there is any word on finding Momentum or the Multi-tenancy plugin?

**Reply by MaqAnquor** [reply]
No sorry, if you find do share with me

> @nightcrawler123 Thank you kindly. Do you know if there is any word on finding Momentum or the Multi-tenancy plugin?

**Reply by drbyrd** [reply]
We are excited to announce the launch of the WPCD Community Discord server! This server is dedicated to continuing the work of Nigel and supporting the WPCD community. Join us to collaborate, share knowledge, and contribute to the development of WPCD.

To join, click on the invite link: https://discord.gg/vSEuN9hD

Once inside, please read the rules and introduce yourself. We look forward to seeing you there and building a strong, supportive community together!

**Reply by drbyrd** [reply]
@nightcrawler123 also, would you mind editing my email out of that quote. I shouldn't have put that out there m8... Thanks!

**Comment by jaiivarsson** [comment]
Anyone know if the FB group is still going? Was Nigel the only admin of the group? I made a request to join but only after the main WPCD site went down.

**Reply by ghost** [reply]
Its all over now with Nigel's sad demice. He was One man army - With him all his works Website, FB Group and Development of the project have stopped

**Reply by qvdrei** [reply]
It's truly amazing what he achieved alone. That's a good thing but also a bad thing. He must have died quick (probably a car accident or something related to the heart). He didn't even have time to pass the info to someone he can trust.

**Reply by jaiivarsson** [reply]
Given some of the support style messages that have already started in these threads, what do people think about us setting up a discord or something similar to keep a community going?

**Reply by drbyrd** [reply]
@jaiivarsson  I'd love to help with that! Please reach out to drbyrd(at)gmail(dot)com

**Comment by eslam-dev** [comment]
The news of NIGEL passing came as a shock to all of us.
 He contributed significantly to the development of WPCD. 
Therefore, I have decided to continue the updates for WPCD on the following branch, and I will be uploading the complete documentation soon.
https://github.com/developvi/wp-cloud-deploy

**Reply by jaiivarsson** [reply]
Let us know if there is anything we can do to help

**Reply by eslam-dev** [reply]
@jaiivarsson 
Thank you for offering your help. If we need any support, we will definitely reach out to you. Currently, I need assistance in gathering suggestions and analyzing issues occurring in the plugin.
